modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -52,7 +54,7 @@ def validate_request!(validator_class)
       #
       # @return [ClaimsApi::Veteran] Veteran to act on
       def target_veteran
-        @target_veteran ||= if @is_valid_ccg_flow
+        @target_veteran ||= if @is_token_valid
```

Comment:
Can we change the instance variable name to `@is_valid_ccg_token`?
`@is_token_valid` is just vague enough to be confusing here.
Because on this `if`, we're wanting it to only be executed if the token received is specifically a valid ccg token.
The `elsif` and `else` also require valid tokens, but in their cases, they're not ccg tokens.

So I think changing the name to `@is_valid_ccg_token` will help make it much more clear what's happening here.

Reply:
Makes sense, v1 uses `@is_token_valid` so I figure I'd keep it as such in case it's used elsewhere that I don't know, but I don't see it used anywhere else, so I'll update it to `@is_valid_ccg_token`